### PR TITLE
Create models, add migration and seeders for all tables

### PR DESCRIPTION
**Pull request by atanackoo21**
I created all models, migrations and seeders which i identified that our system should have.

### CODE REVIEW

File: src/database/migrations/20191126082619-create-volunteer.js [hunk 1]
```diff
@@ -16,10 +16,11 @@ module.exports = {
         type: Sequelize.STRING,
         allowNull: false
       },
-      created_at: { 
-        type: Sequelize.DATE,
+      created_at: {
+        type: 'TIMESTAMP',
+        defaultValue: new Date(	'03/25/2015'),
```
Comment (coka) [inline src/database/migrations/20191126082619-create-volunteer.js]: Why is this hardcoded? Also, why do we have this whitespace?

File: src/database/migrations/20191126082619-create-volunteer.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.createTable("Volunteers", {
-      volunteer_id: {
+      id: {
```
Comment (coka) [inline src/database/migrations/20191126082619-create-volunteer.js]: Don't modify existing migrations.

File: src/database/migrations/20191206105932-create-call-description.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        key: 'id',
+        },
+        allowNull: false
+      },
+      suicide_factor_id: {
+        type: Sequelize.INTEGER,
+        references: {
+        // This is a reference to another model
+        model: 'Types_of_suicide_factors',
+        // This is the column name of the referenced model
```
Comment (coka) [inline src/database/migrations/20191206105932-create-call-description.js]: These comments are not needed.

File: src/database/migrations/20191206105932-create-call-description.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      },
+      content: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      state_after_call: {
+        type: Sequelize.ENUM('bolje',
+                              'isto (nepromenjeno)',
+                              'gore',
+                              'neutvrđeno'),
```
Comment (coka) [inline src/database/migrations/20191206105932-create-call-description.js]: Don't use 🇷🇸 in the DB.

File: src/database/migrations/20191206111012-create-call.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        defaultValue: new Date('03/25/2015'),
+        allowNull: false
+      }
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable("Calls");
+  }
+};
```
Comment (coka) [inline src/database/migrations/20191206111012-create-call.js]: Define everything in a single migration. I can explain why this is important during standup.

File: src/database/models/call.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+          defaultValue: sequelize.literal('CURRENT_TIMESTAMP'),
+          allowNull: false
+        }    
+    }, 
+    {
+        sequelize,
+        modelName: 'Call',
+        timestamps: false
+    });
+}
```
Comment (coka) [inline src/database/models/call.js]: "No newline at end of file"

Please fix this for all files.

**Review by coka — CHANGES_REQUESTED** [review]
For the first pass, these are the things we should be aware of.

File: src/database/migrations/20191206111012-create-call.js (hunk 5 above)
Comment (atanackoo21) [reply src/database/migrations/20191206111012-create-call.js]: It means that migrations for all tables need to put in one migrations file?

Comment (coka) [reply src/database/migrations/20191206111012-create-call.js]: Yes.

**Review by coka — CHANGES_REQUESTED** [review]
There's still a lot of work to be done here. I would close this PR (it will still be around for future reference), because I don't think we'll have time to fix everything today. We can discuss live.

**Pull request closed without merging**